reliabot/reliabot.py

```diff
@@ -41,10 +41,21 @@
 from typing import TextIO
 from typing import Union
 
-from ruamel.yaml import YAML  # ruamel.yaml preserves comments, PyYAML doesn't.
-from ruamel.yaml.comments import CommentedMap
-from ruamel.yaml.comments import CommentedSeq
-from ruamel.yaml.parser import ParserError
+try:
+    # ruamel.yaml preserves comments, PyYAML doesn't.
+    from ruamel.yaml import YAML
+    from ruamel.yaml.comments import CommentedMap
+    from ruamel.yaml.comments import CommentedSeq
+    from ruamel.yaml.parser import ParserError
+except ModuleNotFoundError, ImportError as e:
+    error_message = ''.join([
+        f"{e}",
```

Comment:
`ModuleNotFoundError` is a subclass of `ImportError`, so there's no need to catch both types. The question is whether directing end-users to the installation instructions is likely to be more helpful for the cases of `ImportError` that are _not_ `ModuleNotFoundError` instances.

Based on this [StackOverflow post](https://stackoverflow.com/a/77361812/18829), I think for things like circular imports or errors in the `ruamel.yaml` module itself, the specific exception details would be more useful, as the fix is likely to require code changes, not just proper installation.

Note also in the [pre-commit.ci checks that failed](https://results.pre-commit.ci/run/github/731308823/1704819372.r1pOdxCDRGm3DFCTvEswkA) that you used the Python 2 syntax for multiple exception classes, in Python 3 these need to be parenthesized. It's also helpful to use a more meaningful name for the exception vairable.

```suggestion
except ModuleNotFoundError as module_not_found:
    error_message = ''.join([
        f"{module_not_found}",
```